build.sbt

```diff
@@ -124,3 +124,5 @@ ThisBuild / developers := List(
   Developer("joroKr21", "Georgi Krastev", "joro.kr.21@gmail.com", url("https://twitter.com/Joro_Kr")),
   Developer("TimWSpence", "Tim Spence", "timothywspence@gmail.com", url("https://twitter.com/timwspence"))
 )
+
+lazy val docs = project.in(file("site")).enablePlugins(TypelevelSitePlugin)
```

Comment:
Why `in(file("site"))`?

Reply:
It can be anything you want except `docs/`, per mdoc instructions.
https://scalameta.org/mdoc/docs/installation.html#sbt

Reply:
Note that the docs do still live in the `docs/` folder.